packages/opentelemetry-api/src/metrics/BoundInstrument.ts

```diff
@@ -44,3 +45,14 @@ export interface BoundMeasure {
     spanContext: SpanContext
   ): void;
 }
+
+/** Base interface for the Observer metrics. */
+export interface BoundObserver {
+  /**
+   * Sets callback for the observer. The callback is called once and then it
```

Comment:
This is slightly confusing to me. Why do you not just call observe multiple times to observe multiple values? Seems like a lot of indirection going on here.

Why do we have this:

```typescript
cpu.setCallback((observerResult) => {
  observerResult.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '1' }));
  observerResult.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '2' }));
  observerResult.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '3' }));
  observerResult.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '4' }));
});
```

and not this:

```typescript
cpu.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '1' }));
cpu.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '2' }));
cpu.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '3' }));
cpu.observe(getCpuUsage, meter.labels({ pid: process.pid, core: '4' }));
```

Reply:
This is what spec wanted this to be:
https://github.com/open-telemetry/oteps/blob/master/text/0072-metric-observer.md
So they specified this precisely that first it should setCallback and then you can call observe.
`setCallback` has been even added much earlier when `NoopMetric` has been created